Adding definition api endpoint test and changing json file dir

suheb opened the pull request:
Signed-off-by: Suhaib Khan suheb.work@gmail.com

suheb committed:
Adding definition api endpoint test and changing json file dir

Signed-off-by: Suhaib Khan <suheb.work@gmail.com>

kylepjohnson commented:
Hey @suheb I don't know what exactly you're trying to do here, so I'm going to reject this.

Would you mind testing locally and seeing what's up?

suheb committed:
Adding test to see if treebank corpora exists

suheb commented:
@kylepjohnson Sorry about that. I added the test for definition endpoint but forgot to check if the treebank corpora exist. I've updated test.py now.
